src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonMetadataServices.cs

```diff
@@ -91,15 +91,17 @@ public static partial class JsonMetadataServices
         /// <param name="options"></param>
         /// <param name="createObjectFunc"></param>
         /// <param name="propInitFunc"></param>
+        /// <param name="serializeFunc"></param>
```

Comment:
Why aren't we documenting these parameters? This is public API.

Reply:
Fixed.